package.json

```diff
@@ -70,7 +72,7 @@
     "clean": "gts clean",
     "compile": "tsc -p . && cp src/*.json build/src && cp src/*.js build/src",
     "fix": "gts fix && eslint --fix samples/*.js samples/**/*.js",
-    "prepare": "npm run compile",
+    "prepublish": "npm run compile && node ./build/tools/prepublish.js",
```

Comment:
`npm` shows warnings:
```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
```
Can you change it to `prepublishOnly`?

Reply:
(or `prepare`, I'm not sure which one will actually work here)

Reply:
`prepare` and `prepublishOnly` are not recognized in old NPM versions. This resulted in a build break in the Node 6 CI job at one point. I'm going to see if I can work around this by calling `prepare` from the `pretest` target. The downside is on new NPM versions, the `prepare` will run twice -- once during `install` and once during `test`.